### PR TITLE
updated form.ts, form.html, resolves 374

### PR DESCRIPTION
**Pull request by DKJackson**

### CODE REVIEW

File: src/components/form.ts [hunk 1]
```diff
@@ -1,48 +1,31 @@
...
-    inputs: ['config']
+    styleUrls: ['../styles/form.css']
 })
+
 export class BossyForm {
-    configKeys: Array<string>;
-    configForm: FormGroup;
-    @Input() config: any;
+    bossyForm: FormGroup;
+    @Input() config: BossyFormConfig;
```
Comment (JasonGroven) [inline src/components/form.ts]: Should this be` @Input('config') config: any;`?  I was getting errors trying to specify the config type in this manner, before.

Comment (tabersays) [inline src/components/form.ts]: It should not be any, what errors are you getting?

Comment (DKJackson) [inline src/components/form.ts]: The BossyFormConfig class is in the config folder.  Did you pull that?

Comment (DKJackson) [inline src/components/form.ts]: I added a name field to the BossyFormInputConfig, but it looks like I forgot to initialize it in the constructor.  Could that be the problem?

Comment (DKJackson) [inline src/components/form.ts]: Come to think of it, I don't see a reason that the input should need to know the name.  That seems to be solely for the sake of creating the form control which is handled by the FormBuilder in the BossyForm component.

**Comment by glawson** [comment]
@DKJackson - please fix merge conflicts.

**Pull request merged**